### PR TITLE
LPD-18577 Create GroovyScriptUses Modal

### PR DESCRIPTION
**Pull request by danvb112**
related Issue: https://liferay.atlassian.net/browse/LPD-18577

### CODE REVIEW

**Comment by danvb112** [comment]
Hey @igor-franca, this PR is blocked because it depends on https://github.com/liferay-workflow/liferay-portal/pull/1093 and https://github.com/liferay-objects/liferay-portal/pull/3148

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/GroovyScriptUsesModal.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+					<Text color="secondary">
+						{Liferay.Language.get(
+							'resolve-all-active-scripting-uses-before-proceeding'
+						)}
+					</Text>
+
+					<Text color="secondary">
+						{Liferay.Language.get(
+							'these-active-scripting-uses-must-be-resolved-click-to-open-a-new-tab'
+						)}
```
Comment (igor-franca) [inline modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/GroovyScriptUsesModal.tsx:60]: Hi @danvb112, I noticed that this modal in Figma can show the sentence in a single line:
 
In my tests I can see a subtle difference between Figma and the implemented modal. We have a line break here, which for me takes away some of the elegance of the modal. I had the opportunity to consult Júlia and she approved the suggestion of using the lg size property in this modal. 

script-modal

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @danvb112, just a little suggestion 😄

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/GroovyScriptUsesModal.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/GroovyScriptUsesModal.tsx:60]: Hey @igor-franca, thanks for noticing this, I've changed the modal size! :smile:

**Comment by danvb112** [comment]
ci:test:sf

**Review by igor-franca — APPROVED** [review]

**Pull request closed without merging**